lte/gateway/c/li_agent/src/BUILD.bazel

```diff
@@ -25,6 +30,36 @@ cc_binary(
     ],
 )
 
+download_pkgs(
+    name = "liagent_requisites",
+    image_tar = "@ubuntu_focal//image",
+    packages = [
```

Comment:
The required dependencies can by found with `ldd` or one-by-one by simply executing the binary.